content/english/secret-messages/using-repl-it.md

```diff
@@ -5,6 +5,8 @@ weight: 1
 draft: false
 ---
 
+{{< youtube sdO27-l4aqM >}}
```

Comment:
<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/sdO27-l4aqM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>